### PR TITLE
README files for Feed Ingestion and Video Handlers

### PR DESCRIPTION
**Pull request by jsutterfield**

### CODE REVIEW

**Comment by armonr** [comment]
/s/wiki/wikia :)

File: extensions/wikia/VideoHandlers/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+from there.
+
+Here's a document on Wikia One which has a diagram for the overall process:
+[Feed Ingestion Architecture]( https://one.wikia-inc.com/wiki/Video/Technical_Documentation/Feed_Ingestion_Architecture?file=Ingestion.jpg)
+
+## Video Handlers
+A first thing to note is that Wikia does not store any videos on our servers. Instead we store metadata about the video in
+the video151 database (the database for video.wikia.com). The exact tables which we store information in can be found in
+the [Feed Ingestion Architecture](https://one.wikia-inc.com/wiki/Video/Technical_Documentation/Feed_Ingestion_Architecture?file=Ingestion.jpg) link.
+Because we don't have the physical files ourselves, we have VideoHandler classes which are in charge of determining how to
```
Comment (armonr) [inline extensions/wikia/VideoHandlers/README.md]: Maybe remove the query string in the URL

**Comment by armonr** [comment]
:+1:

File: extensions/wikia/VideoHandlers/README.md (hunk 1 above)
Comment (jsutterfield) [inline extensions/wikia/VideoHandlers/README.md]: cool call!

File: extensions/wikia/VideoHandlers/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Once those videos have been added to video wiki, users can then embed them in their local wikias (eg, thelastofus.wikia.com,
+callofduty.wikia.com, disney.wikia.com, etc). The FeedIngester classes found in this extension are part of the ETL process
+for each of these providers. Each one will check its corresponding provider's api, then normalize the data before uploading
+it onto our site.
+
+The cron job which runs is the [ingestPartnerWithVideoData.php script](https://github.com/Wikia/app/blob/dev/maintenance/wikia/ingestPartnerVideoWithData.php).
+It iterates over our list of currently active premium providers, instantiates the corresponding feedingester class for
+the provider, and kicks off the ingestion process.
+
+Within premium videos there's a distinction between whether a video is a remote asset or not. A video which is not a
```
Comment (lizlux) [inline extensions/wikia/VideoHandlers/README.md]: Maybe add a heading here: "Remote assets"

Comment (jsutterfield) [inline extensions/wikia/VideoHandlers/README.md]: Good idea.

File: extensions/wikia/VideoHandlers/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+the video151 database (the database for video.wikia.com). The exact tables which we store information in can be found in
+the [Feed Ingestion Architecture](https://one.wikia-inc.com/wiki/Video/Technical_Documentation/Feed_Ingestion_Architecture) link.
+Because we don't have the physical files ourselves, we have VideoHandler classes which are in charge of determining how to
+play the video based on its provider. In some cases this involves loading a javascript player (as is the case with videos
+ingested from Ooyala), and in others it involves just using an iframe (as is the case for videos ingested from Maker Studios).
+
+VideoHandlers are registered to their corresponding video files via the `$wgMediaHandlers[]` array (eg, `$wgMediaHandlers['video/makerstudios'] = 'MakerstudiosVideoHandler';`).
+Mediawiki keys of the img_major_mime and the img_minor_mime columns found in the image table joined by a '/' to
+determine which key to use when checking `$wgMediaHandlers`. So, for example, all Maker Studios videos have 'video' as
+the value for their img_major_mime, and 'makerstudios' as the value for their img_minor_mime. Mediawiki will join those
```
Comment (lizlux) [inline extensions/wikia/VideoHandlers/README.md]: maybe wrap `img_minor_mime` with the ` `

**Comment by lizlux** [comment]
Just a couple suggestions. LGTM!

**Pull request merged**